src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -346,27 +694,35 @@ public with sharing class AFFL_MultiRecordType_TEST {
         insert contact;
 
         //Craete account of Business Organization record type
-        Account acc = new Account(Name='Acme', RecordTypeId = orgRecTypeID);
+        Account acc = new Account(Name='Acme', RecordTypeId = UTIL_Describe_API.getBizAccRecTypeID());
         insert acc;
 
         //Create Business affiliation
         Affiliation__c affl = new Affiliation__c(Contact__c = contact.ID, Account__c = acc.ID, Role__c = 'Student', Status__c = 'Current');
+
         Test.startTest();
-        AFFL_ContactAccChange_TEST.resetAfflFlags();
-        insert affl;
+            AFFL_ContactAccChange_TEST.resetAfflFlags();
+            insert affl;
         Test.stopTest();
         //Verify Progran Enrollment record has been created
-        List<Program_Enrollment__c> programEnrollments = [select ID, Contact__c from Program_Enrollment__c where Affiliation__c = :affl.ID];
+        List<Program_Enrollment__c> programEnrollments = [SELECT ID, Contact__c FROM Program_Enrollment__c WHERE Affiliation__c = :affl.ID];
         System.assertEquals(1, programEnrollments.size());
         System.assertEquals(contact.ID, programEnrollments[0].Contact__c);
     }
 
     @isTest
     private static void autoCreateProgramEnrollmentDifferentRole() {
-        AFFL_MultiRecordType_TEST.setup();
+        List<AffiliationMappingWrapper> affiliationMappingWrapperList =
+            AFFL_MultiRecordType_TEST.setupMappingsByNameAndLabel();
+        String bizOrgRecordTypeName = affiliationMappingWrapperList[1].getAccountRecordTypeName();
 
         //Turn on Academic Record auto-creation for Business Organization mapping - doesn't make a lot of sense for this type, but just for testing
-        Affl_Mappings__c bizOrgMapping = [select Auto_Program_Enrollment__c from Affl_Mappings__c where Name = 'Business Organization'];
+        Affl_Mappings__c bizOrgMapping = [SELECT Auto_Program_Enrollment__c,
+            Auto_Program_Enrollment_Role__c,
+            Auto_Program_Enrollment_Status__c
+            FROM Affl_Mappings__c
+            WHERE Name = :bizOrgRecordTypeName];
```

Comment:
Name is a potentially encryptable field, this will break shield.

Reply:
This is a custom setting, but we have a better way to get this anyway since we already have it stored in the mapping wrapper model.